mw-webapp/src/pages/authPage/AuthPage.tsx

```diff
@@ -1,17 +1,38 @@
-import {Link} from "react-router-dom";
-import {useAuth} from "src/utils/useAuth";
+import {User} from "firebase/auth";
+import {useEffect, useState} from "react";
+import {Link, Navigate} from "react-router-dom";
+import {Button} from "src/component/button/Button";
+import {handleUserCredentials} from "src/utils/auth/handleUserCredentials";
+import {handleUserAuthState} from "src/utils/auth/handleUserAuthState";
+import {handleLogIn} from "src/utils/auth/handleLogIn";
+import {handleLogOut} from "src/utils/auth/handleLogOut";
+
+const BUTTON_LOG_IN_VALUE = "Login";
+const BUTTON_LOG_OUT_VALUE = "Logout";
+const LINK_TEXT = "Workflow";
+const MAIN_PATH = "/main";
```

Comment:
we have pages.tsx file with router related data. remember about encapsulation.

Reply:
Use path from pages.tsx file
By the way, the paths there are specified as strings, not as constants